db/migrate/20250624062003_add_type_to_container_projects.rb

```diff
@@ -0,0 +1,5 @@
+class AddTypeToContainerProjects < ActiveRecord::Migration[7.1]
+  def change
+    add_column :container_projects, :type, :string
+  end
```

Comment:
We'll also want an index on this column
```suggestion
    add_index :container_projects, :type
  end
```